department-of-veterans-affairs/caseflow
# Judge check out | Send to next team from their queue

Connects #5149 

### Description
Adds an Action dropdown to `JudgeReviewTaskTable`.

### Acceptance Criteria 
- [ ] Action column/dropdown present in `JudgeReviewTaskTable`, hidden behind `:judge_assignment` FeatureToggle

### Testing Plan
1. Go to `/queue` as `BVAAABSHIRE` / a judge, confirm Action column not visible
2. Enable `:judge_assignment` FeatureToggle, reload page, confirm Action column visible

![screen shot 2018-05-16 at 4 58 09 pm](https://user-images.githubusercontent.com/8533004/40143767-53cd1d22-592a-11e8-9b54-2503ded48644.png)
